Modelica/package.mo

```diff
@@ -2278,7 +2278,7 @@ Graphical illustrations shall not be added in the diagram layer, but can be adde
 </html>"));
     end Icons;
     annotation (DocumentationClass=true,Documentation(info="<html>
-<p>A Modelica main package should be compliant with the UsersGuide stated in this documentation:</p>
+<p>Every main package of the Modelica Standard Library should be compliant with the UsersGuide stated in this documentation:</p>
```

Comment:
To me this seems to highlight the confusion/ambiguity:
- Are there any non-main packages in MSL that can ignore the conventions? 
- I interpreted "main package" as including all packages in this repo, Modelica, ModelicaTest, Complex. If so "of the MSL" seems too restrictive, and we should state that all packages of the repo, or it may be that the intent was that some minor packages of the repo like ObsoleteModelica4 can ignore it - if so we should state that.

Reply:
Indeed, it would be very good to get this clarified.

Reply:
Remove "ModelicaTest" - since there is no dicumentation. Better (I would even omit parentheses): 
"Every main package, such as Modelica, Complex or ModelicaServices, released by..."

Reply:
With only examples of "main packages" from the MSL, I find the statement somewhat ambiguous, as one may think that at the time of writing all "main packages" released by the MSL were assumed to be part of MSL.  Isn't there some better place for rules applying to all libraries released by the Modelica Association?

Would a first step of clarification be to at least include the examples of "main package"?
```suggestion
<p>Every main package of the Modelica Standard Library (such as Modelica, Complex, ModelicaTest) should be compliant with the UsersGuide stated in this documentation:</p>
```

Reply:
@tobolar isn't it a problem with your suggestion that ModelicaTest actually is a "main package"?  Don't we need to drop _every_ for this to make sense?
```suggestion
<p>User-facting main packages of the Modelica Standard Library (such as Modelica and Complex) should be compliant with the UsersGuide stated in this documentation:</p>
```

Reply:
The question is who is a recipient of the comment. If it is a user, then I simply pointed out the ModelicaTest is not part of the official MSL release.
If it is a developer, well then you are right - this really doesn't matter.